code/modules/projectiles/guns/launcher/grenade_launcher.dm

```diff
@@ -174,6 +174,7 @@
 	force = WEAPON_FORCE_PAINFUL
 	matter = list(MATERIAL_STEEL = 20, MATERIAL_WOOD = 10)
 	force = 5
+	price_tag = 500
```

Comment:
why?

Reply:
because the default price is 0, there was no override, and the minimum price for the first option, let alone the rest, is 100.
since the rest of the makeshift guns have a price tag #of 500, and otherwise the makeshift grenade launcher would not be eligible for the gun assembly, I made the price 500.

> 
> 
> why?

because the minimum price for the first option is 100, scaling per option, the default price is 0, and all the other makeshift guns have a price tag of 500; otherwise, the makeshift grenade launcher would not be a possibility for gun assemblies.